ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/WriteService.scala

```diff
@@ -95,13 +96,17 @@ trait WriteService
     * @param estimatedInterpretationCost Estimated cost of interpretation that may be used for
     *                                    handling submitted transactions differently.
     * @param telemetryContext            Implicit context for tracing.
+    * @param loggingContext            Implicit context for logging.
     */
   def submitTransaction(
       submitterInfo: SubmitterInfo,
       transactionMeta: TransactionMeta,
       transaction: SubmittedTransaction,
       estimatedInterpretationCost: Long,
-  )(implicit telemetryContext: TelemetryContext): CompletionStage[SubmissionResult]
+  )(implicit
+      telemetryContext: TelemetryContext,
+      loggingContext: LoggingContext,
```

Comment:
@miklos-da, just to make sure, can we do it like this without deprecating the method (like we did for the telemetry context)?

Reply:
Sounds good to me.

Reply:
Will do so, even if not used. 
The reason I did not do it from the first is that we have the `submitTransaction` as part of the WriteService definition, but all of the other methods have their own trait. 
I will add the logging context in this PR, and I'm also thinking of extracting `submitTransaction` into its own trait as well, to be consistent in usages.